src/Textarea/Textarea.component.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+type Props = {
+  /**
+   * Select Textarea Size
+   *
+   * @default 'md'
+   **/
+  TextareaSize: string;
```

Comment:
Let's do `textareaSize` here. Capital first letter is generally reserved for Components in React.

Reply:
Done!